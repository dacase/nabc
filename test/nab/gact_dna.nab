#include "xmin_opt.h"
struct xmin_opt xo;

molecule m;
float 	m_xyz[6000], f[6000];
float 	dgrad, fret, grms;
int 	ier; 

xmin_opt_init( xo );

m = getpdb("gact_dna.pdb");
readparm(m, "gact_dna.top");
mm_options( "ntpr=1, gb=1, kappa=0., rgbmax=99.0, cut=99.0, diel=C ");
getxv( "gact_dna.xmin1.x", m.natoms, fret, m_xyz, f );
mme_init( m, NULL, "::Z", m_xyz, NULL);

xo.grms_tol =0.000001;

fret = mme( m_xyz, f, -1 );
fret = xmin( m.natoms, m_xyz, f, fret, grms, xo );

newton(m_xyz, 3*m.natoms, fret, mme, mme2, 0.00000000001, 0.0, 8);

nmode( m_xyz, 3*m.natoms,mme2, 0.0,0.0,0);

mme2_timer();
