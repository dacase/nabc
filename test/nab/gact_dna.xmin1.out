Reading parm file (gact_dna.top)
title:
                                                                                
	mm_options:  ntpr=10
	mm_options:  gb=1
	mm_options:  kappa=0.
	mm_options:  rgbmax=99.0
	mm_options:  cut=99.0
	mm_options:  diel=C
no velocities were found
     bond:    238.139091157
    angle:    545.091945415
 dihedral:    642.698273678
    enb14:    258.194304398
    eel14:  -3289.786648713
      enb:   -516.638972877
      eel:   5182.543981893
      egb:  -8718.330315850
    econs:      0.000000000
    esurf:      0.000000000
    Total:  -5658.088340898
     frms:     20.046474008
ff:    10  -6453.89    735.19   -341.58   1857.49      0.00  -8704.99  7.67e-01
ff:    10  -6454.09    734.89   -341.69   1858.18      0.00  -8705.47  3.81e-01
ff:    20  -6462.16    729.33   -340.30   1840.54      0.00  -8691.74  1.29e-01
ff:    30  -6466.30    730.00   -341.26   1808.96      0.00  -8664.00  1.07e-01
ff:    40  -6467.52    729.89   -343.97   1847.16      0.00  -8700.60  6.98e-02
ff:    50  -6468.09    729.69   -344.90   1870.36      0.00  -8723.25  1.89e-02
ff:    60  -6468.24    729.49   -345.83   1880.86      0.00  -8732.77  1.42e-02
ff:    70  -6468.32    729.61   -345.72   1874.44      0.00  -8726.65  8.23e-03
ff:    80  -6468.35    729.54   -345.55   1872.35      0.00  -8724.68  5.35e-03
ff:    90  -6468.36    729.67   -345.65   1872.36      0.00  -8724.74  3.04e-03
ff:   100  -6468.37    729.63   -345.67   1873.96      0.00  -8726.29  1.93e-03
ff:   110  -6468.37    729.60   -345.54   1872.98      0.00  -8725.40  8.25e-04
ff:   120  -6468.37    729.60   -345.53   1872.27      0.00  -8724.71  6.32e-04
ff:   130  -6468.37    729.60   -345.51   1872.08      0.00  -8724.53  1.64e-04
ff:   140  -6468.37    729.60   -345.52   1872.17      0.00  -8724.62  1.94e-04
ff:   150  -6468.37    729.59   -345.51   1872.26      0.00  -8724.71  5.73e-05
ff:   160  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  3.09e-05
ff:   170  -6468.37    729.59   -345.51   1872.22      0.00  -8724.67  9.69e-06
ff:   180  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  4.05e-06
ff:   190  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  2.08e-06
ff:   200  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  5.03e-07
ff:   210  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  4.54e-07
ff:   220  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  2.80e-07
ff:   230  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.19e-07
ff:   240  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  4.03e-08
ff:   250  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.94e-08
ff:   260  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.79e-08
ff:   270  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.01e-08
ff:   280  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  6.46e-09
ff:   290  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  2.86e-09
ff:   300  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  9.45e-10
ff:   310  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  3.67e-10
ff:   320  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.51e-10
ff:   330  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.66e-10
ff:   340  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  4.01e-11
ff:   350  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.94e-11
ff:   360  -6468.37    729.59   -345.51   1872.23      0.00  -8724.68  1.02e-11
     bond:     44.558541656
    angle:    144.567427903
 dihedral:    540.467273419
    enb14:    213.968326108
    eel14:  -3262.979506382
      enb:   -559.480842305
      eel:   5135.208956694
      egb:  -8724.682224002
    econs:      0.000000000
    esurf:      0.000000000
    Total:  -6468.372046908
     frms:      0.000000000
